tests/views_integration/Tribe/Events/Views/V2/Partials/Month_View/Calendar_Body/Day/Calendar_Events/Calendar_Event/__snapshots__/TooltipTest__test_render_with_event_with_featured_image_with_description_without_cost__1.php

```diff
@@ -22,8 +22,6 @@ class="tribe-common-c-image__bg"
 		</div>
 	</a>
 </div>
-		<div class="tribe-events-calendar-month__calendar-event-tooltip-description tribe-common-b3">
```

Comment:
this test says it is rendered with description, but the snapshot does not have a description. can we fix this test to make sure we're outputting the correct snapshot?

Reply:
Thanks! Haven't worked on these test, they were already there. It seems that the way they were setting the content wasn't actually setting it. Will fix them.